apps/src/code-studio/header.js

```diff
@@ -108,10 +112,6 @@ header.build = function (scriptData, stageData, progressData, currentLevelId, pu
     e.stopPropagation();
     $('.header_popup').is(':visible') ? hideHeaderPopup() : showHeaderPopup();
   });
-  $('.header_popup').click(function (e) {
-    e.stopPropagation(); // Clicks inside the popup shouldn't close it
-  });
```

Comment:
This was making it so that all clicks inside of the popup were ignored, and I couldn't use the toggle. Instead I moved this logic into hideHeaderPopup.